src/main/scala/model/entities/Enemy.scala

```diff
@@ -41,14 +42,52 @@ abstract class Enemy(override val position: Position,
 
 
 /**
- * Basic Zombie Enemy.
+ * Zombie is the base enemy of the game.
+ *
+ * @param position the position in which the zombie is placed.
+ * @param life the life that the zombie currently has.
+ * @param state the state of the zombie.
+ */
+case class Zombie(override val position: Position = (Random.between(0, NumOfLanes), LanesLength + Random.between(0, 20)),
```

Comment:
Consider putting all these magic numbers in the DefaultValues file.

Reply:
yess